modules/gapi/include/opencv2/gapi/streaming/gstreamer/gstreamerpipeline.hpp

```diff
@@ -40,6 +40,15 @@ class GAPI_EXPORTS GStreamerPipeline
 
 using GStreamerPipeline = gst::GStreamerPipeline;
 
+// NB: Function for using from python
+GAPI_EXPORTS_W cv::Ptr<IStreamSource>
+inline get_streaming_source(cv::Ptr<GStreamerPipeline>& pipeline,
```

Comment:
Why it's needed?

Reply:
I did try to wrap just the method `GStreamerPipeline::getStreamingSource()` - but didn't succeed. The problem is in the returning value

I can try to find better solution (probably, wrap the method somehow)

Reply:
rewritten wrapping: now it is a class method, not a separate function.
More convenient to use:
```
pipeline = """some ! gst ! pipeline ! appsink name=sink1
              some ! gst ! pipeline ! appsink name=sink2"""
pp = cv.gapi.wip.GStreamerPipeline(pipeline)
src1 = pp.get_streaming_source("sink1")   # instead of src1 = cv.gapi.wip.get_streaming_source(pp, "sink1")
src2 = pp.get_streaming_source("sink2")
...
ccomp = c.compileStreaming()
ccomp.setSource(cv.gin(src1, src2))
...
```

Reply:
I had problems with wrapping the method as it's compllicated to wrap the returning class `IStreamSource::Ptr = std::shared_ptr<IStreamSource>`

set to have a separate functions for now

left a comment about it